mindjunkies/courses/migrations/0003_rename_user_coursetoken_teacher_and_more.py

```
@@ -0,0 +1,34 @@
+            old_name='user',
+            new_name='teacher',
+        ),
+        migrations.AddField(
+            model_name='coursetoken',
+            name='motivation',
+            field=models.TextField(default='', help_text='Write your motivation or plan for the course, including how you plan to organize it.'),
+        ),
+        migrations.AlterField(
+            model_name='coursetoken',
+            name='course',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tokens', to='courses.course'),
```

Ruff (0.8.2)

Line too long (125 > 119)

(E501)